readthedocs/oauth/models.py

```diff
@@ -206,3 +210,30 @@ def matches(self, user):
                 },
             ),
         } for project in projects]
+
+
+class RemoteRelation(TimeStampedModel):
```

Comment:
We may need to rename this model to something like `RemoteRepositoryRelation` because we will also need a relation model for `RemoteOrganization` as well and we will have conflicting names.

Does it make sense to add make `remoterepository` field nullable and add `remoteorganization` field as well to this `RemoteRelation` model and use it for both? It seems it's exactly the same data but `RemoteOrganization` does not has `admin` field.